tutorials/containers/introduction-to-lxd-projects/introduction-to-lxd-projects.md

```diff
@@ -0,0 +1,417 @@
+---
+id: introduction-to-lxd-projects
+summary: An introduction to the projects feature in LXD
+categories: containers
+tags: tutorial,desktop,server,lxd,lxc,projects
+difficulty: 3
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-04-24
+author: Thomas Parrott <thomas.parrott@canonical.com>
+
+---
+
+# Introduction to LXD projects
+
+## Overview
+Duration: 5:00
+
+[LXD](https://linuxcontainers.org/lxd) is a container hypervisor providing a ReST API to manage LXC containers.
+It provides a virtual machine like experience without incurring the overhead of a traditional hypervisor.
+
+
+![logo](images/containers.png)
+
+However when you are managing lots of containers providing different services, it can become confusing to see which containers are dependent on each other.
+
+The projects feature is designed to help in this situation by providing the ability to group one or more containers together into related "projects" that can be used with the `lxc` tool.
+
+This tutorial will show how to create LXD projects, add containers into a project and explore the features that can be specified at a project level.
+
+### Requirements
+
+  - LXD installed and running on Ubuntu 18.04 or higher
+  - You should know how to create and launch an LXD container
```

Comment:
`a LXD container`? (sounds better to me anyway :))

Reply:
@stgraber interesting, I think this one comes down to the different way we pronounce it. 

I've been used to saying "ell ex dee" (i.e spelling it out), and so because the first sound is that of a vowel (L) "an" is correct. 

However as you say "lex dee", then "a" sounds correct.

https://www.quickanddirtytips.com/education/grammar/a-versus-an

This might be a regional thing, or it might be that newcomers spell it out and people more familiar with the project use its shortened version.

What do you think?

Reply:
Ah yeah, that makes sense, the official way to pronounce LXD is lex-dee (per Mark) so then `a` makes sense.